projects/packages/my-jetpack/_inc/components/product-card/recommendation-actions.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button, getProductCheckoutUrl } from '@automattic/jetpack-components';
+import clsx from 'clsx';
+import { useMemo } from 'react';
+import { getMyJetpackWindowInitialState } from '../../data/utils/get-my-jetpack-window-state';
+import useMyJetpackConnection from '../../hooks/use-my-jetpack-connection';
+import styles from './style.module.scss';
+import usePricingData from './use-pricing-data';
+
+const useUpsellLinks = ( slug: string, wpcomProductSlug: string ) => {
+	const { isUserConnected } = useMyJetpackConnection();
+
+	return useMemo( () => {
+		const { myJetpackUrl, siteSuffix } = getMyJetpackWindowInitialState();
+		const purchaseUrl = getProductCheckoutUrl(
+			wpcomProductSlug,
+			siteSuffix,
+			myJetpackUrl,
+			isUserConnected
+		);
+		const interstitialUrl = `#/add-${ slug }`;
+
+		return { purchaseUrl, interstitialUrl };
+	}, [ slug, wpcomProductSlug, isUserConnected ] );
+};
+
+const RecommendationActions = ( { slug }: { slug: string } ) => {
+	const { wpcomProductSlug, learnMoreAction, purchaseAction } = usePricingData( slug );
+
+	const { purchaseUrl, interstitialUrl } = useUpsellLinks( slug, wpcomProductSlug );
+	return (
+		<div className={ styles.actions }>
+			<div className={ clsx( styles.buttons, styles.upsell ) }>
+				{ purchaseAction && (
+					<Button size="small" href={ purchaseUrl }>
+						{ purchaseAction }
+					</Button>
+				) }
+				<Button
+					className={ styles.recommendationLink }
+					size="small"
+					variant="link"
```

Comment:
We don't use the link style on any of the other product cards, is there a reason we want the display to be different on the recommendations? Just to separate them a bit visually from the product cards?

Reply:
There is no reason other than following the design - but indeed we made a decision to just reuse product cards in the end, the initial idea was to create a different design.